hawkular-wildfly-agent/src/main/java/org/hawkular/agent/monitor/extension/MonitorServiceRestartParentAttributeHandler.java

```diff
@@ -54,11 +53,6 @@ protected ServiceName getParentServiceName(PathAddress parentAddress) {
     }
 
     @Override
-    protected boolean isResourceServiceRestartAllowed(OperationContext context, ServiceController<?> service) {
```

Comment:
It is correct but I'm not sure it's acceptable.  It may be OK in the interim but with the PR as it stands if you execute the UpdateCollectionIntervalsCommand it leaves the server in a 'reload-required' state.  That's not really good because in fact the server does not need a reload, we are restarting the agent and that is all that should be required.  I think we can get around this, I'll look at it this morning.  Because no one is calling UpdateCollectionIntervalsCommand at the moment, I don't think it needs to block a merge.